scripts/advancedrum.js

```diff
@@ -0,0 +1,82 @@
+const sampleRUM = hlx.rum.sampleRUM;
```

Comment:
This file is meant to be standalone, so if we decide to go with this approach, it would become its own NPM module and delivered via `https://rum.hlx.page/.rum/@adobe/helix-advanced-rum/index.js` or similar URL.